src/store/wallet.js

```diff
@@ -895,14 +895,14 @@ export default {
     },
 
     createWallet: async function(context, params) {
-      let birthDate = null
+      let birthDate 
 
       const sourceType = params.sourceType
 
       if (sourceType === 'xprv') {
-        birthDate = context.state.birthDate || 0
+        birthDate = { Imported: context.state.birthDate } || null
       } else {
-        birthDate = Number.isInteger(birthDate) ? birthDate : null
+        birthDate = Number.isInteger(birthDate) ? { Imported: birthDate } : 'Current' 
```

Comment:
What happens when you are importing from seed phrase? Is`Current` being used too? That'd be not correct!